### PR TITLE
completed task

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
- [DEMO LINK](https://andriy-shymkiv.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://andriy-shymkiv.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,46 @@
...
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+            href="#laptops&computers"
+            class="nav_link"
+            data-qa="hover"
+            >
```
Comment (illiarerun) [inline src/index.html:34]: will reject next time I see this
```suggestion
            <a
              href="#laptops&computers"
              class="nav_link"
              data-qa="hover"
            >
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,75 @@
...
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+body {
+  margin: 0 auto;
+}
```
Comment (illiarerun) [inline src/style.css:16]: 0 would be enough
```suggestion
body {
  margin: 0;
}
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,75 @@
...
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  background-color: #00acdc;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  top: 56px;
```
Comment (illiarerun) [inline src/style.css:74]: come on, it's easier to use `bottom: 0`
```suggestion
  bottom: 0;
```

**Review by illiarerun — APPROVED** [review]
Good job, left few comments

File: src/index.html (hunk 1 above)
Comment (Andriy-Shymkiv) [reply src/index.html:34]: Okay , got it!

File: src/style.css (hunk 3 above)
Comment (Andriy-Shymkiv) [reply src/style.css:74]: i will remember this rule

**Pull request closed without merging**